packages/jira-adapter/src/adapter.ts

```diff
@@ -644,7 +644,7 @@ export default class JiraAdapter implements AdapterOperations {
     progressReporter.reportProgress({ message: 'Fetching types' })
     const { allTypes: swaggerTypes, parsedConfigs } = await this.getAllTypes(swaggers)
     const userConfigSupportedTypes = this.userConfig.apiDefinitions.supportedTypes
-    const shuldModifySupportedTypes = await getServerInfoTitle(this.client) === JIRA_SERVICE_DESK_FIELD
+    const shuldModifySupportedTypes = !(await hasSoftwareProject(this.client))
```

Comment:
nit: I think that this naming would be clearer
```suggestion
    const shouldOmitBoardSupportedType = !(await hasSoftwareProject(this.client))
```